layouts/shortcodes/fields/new_chapter.html

```diff
@@ -327,3 +341,30 @@
     </div>
   </div>
 </div>
+
+<script>
+  // Variables
+
+  // Location
+  const stateLocationElInputEl = document.getElementById("stateInput");
```

Comment:
If I put the js file in assets/js in which file would I link the script? because if I included it in the content/join/_index.html when the page renders it says that it can't find the js file.

Reply:
If you look up join.js, you'll find that you need to add any new JS files to webpack